<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_5">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>2</number>
      </property>
      <property name="usesScrollButtons">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="infoTab">
       <attribute name="title">
        <string>Справка</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <property name="topMargin">
         <number>9</number>
        </property>
        <item>
         <widget class="QLabel" name="infoLabel">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="styleSheet">
           <string notr="true">font-size: 20px;</string>
          </property>
          <property name="text">
           <string>text</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QPushButton" name="proc_btn">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>150</width>
              <height>0</height>
             </size>
            </property>
            <property name="text">
             <string>Обработать файлы</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="select_btn">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>150</width>
              <height>0</height>
             </size>
            </property>
            <property name="layoutDirection">
             <enum>Qt::LeftToRight</enum>
            </property>
            <property name="text">
             <string>Выбрать папку</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="attrsTab">
       <attribute name="title">
        <string>Столбцы</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QGroupBox" name="planAttrsGB">
          <property name="title">
           <string>План</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <widget class="QCheckBox" name="napr_chb">
             <property name="text">
              <string>Направление</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="cafedra_chb">
             <property name="text">
              <string>Кафедра</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="facultet_chb">
             <property name="text">
              <string>Факультет</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="profile_chb">
             <property name="text">
              <string>Профиль</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="sp_cod_chb">
             <property name="text">
              <string>Код специальности</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="kvalik_chb">
             <property name="text">
              <string>Квалификация</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="edu_form_chb">
             <property name="text">
              <string>Форма обучения</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="start_year_chb">
             <property name="text">
              <string>Год начала</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="standart_chb">
             <property name="text">
              <string>Стандарт ФГОС</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="baza_chb">
             <property name="text">
              <string>База</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="verticalSpacer">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="disciplineAttrsGB">
          <property name="title">
           <string>Дисциплина</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <widget class="QCheckBox" name="in_plan_chb">
             <property name="text">
              <string>В плане</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="ind_chb">
             <property name="text">
              <string>Индекс</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="discipline_chb">
             <property name="text">
              <string>Дисциплина</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="expert_chb">
             <property name="text">
              <string>Экспертное</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="plan_h__chb">
             <property name="text">
              <string>По плану</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="with_teach_h_chb">
             <property name="text">
              <string>С препод.</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="d_ip_h_chb">
             <property name="text">
              <string>ИП</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="d_sr_h_chb">
             <property name="text">
              <string>СР</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="d_patt_chb">
             <property name="text">
              <string>ПАтт</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="req_h_chb">
             <property name="text">
              <string>Обяз. часть</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="not_rq_h_chb">
             <property name="text">
              <string>Вар. часть</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="verticalSpacer_2">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="semesterAttrsGB">
          <property name="title">
           <string>Семестр</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <item>
            <widget class="QCheckBox" name="sem__chb">
             <property name="text">
              <string>Семестр</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="s_itog_h_chb">
             <property name="text">
              <string>Итого</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="s_lek_h_chb">
             <property name="text">
              <string>Лек</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="s_lab_h_chb">
             <property name="text">
              <string>Лаб</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="s_pr_h_chb">
             <property name="text">
              <string>Пр</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="s_krp_h_chb">
             <property name="text">
              <string>КРП</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="s_ip_h_chb">
             <property name="text">
              <string>ИП</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="s_sr_h_chb">
             <property name="text">
              <string>СР</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="s_cons_h_chb">
             <property name="text">
              <string>Конс</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="s_patt_h_chb">
             <property name="text">
              <string>ПАтт</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="control_from_chb">
             <property name="text">
              <string>Форма контроля</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="verticalSpacer_3">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>108</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tableTab">
       <attribute name="title">
        <string>Таблица</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <widget class="QTableWidget" name="tableWidget"/>
        </item>
        <item row="1" column="0">
         <widget class="QPushButton" name="export_btn">
          <property name="text">
           <string>Экспортировать в Excel</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
